build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:1.3.0'
+        classpath 'com.android.tools.build:gradle:2.0.0-alpha1'
```

Comment:
Your Android Studio is recommending it because you are on a pre-release version of Android Studio.